accel-pppd/ipv6/dhcpv6.c

```diff
@@ -831,6 +839,53 @@ static int dhcpv6_read(struct triton_md_handler_t *h)
 	return 0;
 }
 
+static void add_aftr_gw(const char *val)
+{
+	int n = strlen(val);
+	const char *ptr;
+	uint8_t *buf;
+
+	if (!val)
+		return;
+
+	if (val[n - 1] == '.')
+		n++;
+	else
+		n += 2;
+
+	if (n > 255) {
+		log_error("dnsv6: AFTR-Name '%s' is too long\n", val);
+		return;
+	}
+
+	if (!conf_aftr_gw)
+		conf_aftr_gw = _malloc(n);
+	else
+		conf_aftr_gw = _realloc(conf_aftr_gw, conf_aftr_gw_size + n);
+
+	buf = conf_aftr_gw + conf_aftr_gw_size;
+
+	while (1) {
+		ptr = strchr(val, '.');
+		if (!ptr)
+			ptr = strchr(val, 0);
+		if (ptr - val > 63) {
+			log_error("dnsv6: dnssl '%s' is invalid\n", val);
```

Comment:
"dnsv6: dnssl", copy paste error? i guess should be about AFTR-Name

Reply:
Well spotted. I pushed a fix.